packages/x-codemod/tsconfig.json

```diff
@@ -3,5 +3,10 @@
   "compilerOptions": {
     "types": ["react", "mocha", "node"]
   },
-  "include": ["src/**/*", "../../node_modules/@mui/monorepo/test/utils/initMatchers.ts"]
+  "include": [
+    "src/**/*",
+    "../../node_modules/@mui/monorepo/test/utils/initMatchers.ts",
+    "../../node_modules/@mui/material/themeCssVarsAugmentation",
+    "../../node_modules/date-fns"
```

Comment:
I don't know why the typescript on `@mui/x-codemod` complains that `Locale` is not defined in `useMaskedInput` whereas typescript does not complain on `@mui/x-date-icker` itself

So I added is in the tsconfig, but I assume their is a smarter wai of doing that
```
    "../../node_modules/date-fns"
```

Reply:
Hmm... Have you tried adding `"skipLibCheck": true,`? That's one thing that is different between these projects. 🤔

Reply:
Yes, but does not solve the problem